README.md

```diff
@@ -35,6 +35,17 @@ docker run --rm -v "$(pwd)":/code \
 Demo this with `cosmwasm-examples` (going into eg. `erc20` subdir before running),
 with `cosmwasm-plus`, or with a sample app from `cosmwasm-template`.
 
+### Configuring Private Repo Dependencies
+
+SSH credentials for fetching private repo dependencies can be configured through environment variable `GIT_CREDENTIALS` 
+
+```sh
+docker run -e GIT_CREDENTIALS=https://githuser:<TOKEN>@github.com --rm -v "$(pwd)":/code \
+  --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
+  --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
+  cosmwasm/rust-optimizer:0.12.6
+```
```

Comment:
This makes reproducible builds somewhere between hard and impossible. Are you aware of this? This warning should be added to the section here.

Reply:
Can you please explain it a bit more? Should this not depend on how repo dependencies are configured which is equally important for public as well as private repo.